### PR TITLE
Add comments via Mastodon with Emfed?

### PR DESCRIPTION
**Pull request by jamesendreshowell**
Thanks so much for org-static-blog!

This is my _very first_ pull request on GitHub. I am a _complete beginner_ with GitHub, git, and Emacs lisp, so please bear with me and all the mistakes I make in technique and in etiquette! It's a measure of the clarity of the code that I was able to do anything with it.

[Emfed](https://sampsyo.github.io/emfed/) is a very simple way to embed a Mastodon feed. I just [posted an example at my own blog](https://jamesendreshowell.com/2024-08-17-my-first-blog-post.html). That page was produced with the minor changes represented in this pull request.

In short, I added a header #+MASTO-ID: represents the numerical identifier of the Mastodon post. That header gets parsed by 'org-static-blog-get-masto-id and the user's 'org-static-blog-emfed-comments-block gets inserted into the template.

The logic seems trivial, but I'm too new to know if it is submitted correctly in terms of style, or indeed if it is a useful addition to the package. If you find it useful but need me to add or change anything in order to accept it, I'm happy to do more (including changing the documentation and so forth).

Thanks for considering my submission.

### CODE REVIEW

**Comment by bastibe** [comment]
That's really cool, thank you!

I am a bit conflicted about the implementation, though. It seems to me that this should really be part of the `org-static-blog-post-comments` block instead of its own `org-static-blog-emfed-comments-block`. I suppose the reason for using its own block is because it needs that emfed-ID, right?

Does that ID need to be user-generated? Perhaps it could be auto-generated from the URL, with a pinch of Javascript? Then it would all fit neatly into `org-static-blog-post-comments`. That's the way other commenting systems like Disqus work, too.

Or perhaps we should just add an optional generic comment-ID to every post that gets inserted into every comment block, if it specifies a specific placeholder. At any rate, the placeholder should be a bit more specific than `%s`, as that might legitimately occur in a comment block. Perhaps something more like `{{comment-id}}` might be more appropriate.

Let me know what you think about these proposals.

**Comment by jamesendreshowell** [comment]
Wow! Thank you for the instantaneous response!

All your ideas make perfect sense---I regret to admit that I just plain missed incorporating it into the existing comments facility. I'm in the woods away from the computer for a couple days but I'll try to incorporate those suggestions.

**Comment by bastibe** [comment]
No rush whatsoever! Enjoy the woods!

**Comment by jamesendreshowell** [comment]
Sorry for the long-delayed reply.

I hacked around trying to incorporate your suggestions (and in fact used them to make a better design). But after a bit of thought, it seems that adding Emfed integration directly into the `org-static-blog` package itself is probably not the cleanest approach. But thank you for your thoughtful consideration!

**Comment by bastibe** [comment]
Cool! Thank you!
